python/lsst/obs/hsc/hscMapper.py

```diff
@@ -127,7 +139,12 @@ def map(self, datasetType, dataId, write=False):
         """
         copyId = dataId.copy()
         copyId.pop("flags", None)
-        return super(HscMapper, self).map(datasetType, copyId, write=write)
+        location = super(HscMapper, self).map(datasetType, copyId, write=write)
+
+        if 'flags' in dataId:
+            location.getAdditionalData().set('flags', dataId['flags'])
+
+        return location
```

Comment:
This is described as a fix for a bug in the Butler. Does the bug still exist in LSST? If so, do we have a ticket for it?